kms.tf

```diff
@@ -4,6 +4,9 @@ locals {
 
 data "aws_iam_policy_document" "kms_key_global" {
   count = local.create_kms_key
+
+  override_policy_documents = compact([var.kms.additional_policies])
```

Comment:
if we make `var.kms.additional_policies` a list
can we just pass it through like 
```hcl
data "aws_iam_policy_document" "combined" {
  override_policy_documents = [
    data.aws_iam_policy_document.policy_one.json,
    data.aws_iam_policy_document.policy_two.json,
    data.aws_iam_policy_document.policy_three.json
  ]
```